niworkflows/anat/mni.py

```diff
@@ -8,15 +8,20 @@
 import shutil
 import pkg_resources as pkgr
 
+from nilearn.plotting import plot_anat
 from nipype.interfaces.ants.registration import Registration, RegistrationOutputSpec
 from nipype.interfaces.base import (traits, isdefined, BaseInterface, BaseInterfaceInputSpec,
                                     File, InputMultiPath)
 
 from niworkflows.data import getters
+from niworkflows.common import report
 from niworkflows import __packagename__, NIWORKFLOWS_LOG
 
-MAX_RETRIES = 3
-
+DEFAULT_CUTS = {
```

Comment:
This is good for the MNI atlas. It is an atlas precisely because we accurately know where is everything _a priory_.

I agree with you that these defaults would not work, for instance, for showing the result of a segmentation on a brain, since you don't know the coordinates for that case.